From d2545516e1c46ce8a95c4da407f8f6ec0760baab Mon Sep 17 00:00:00 2001
From: naz664 <nazimnavas145@gmail.com>
Date: Tue, 13 Sep 2022 07:48:48 +0000
Subject: [PATCH 2/3] build-Disable-inline-kernel-building

---
 build/tasks/kernel.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/build/tasks/kernel.mk b/build/tasks/kernel.mk
index 363a45bb..161d6137 100644
--- a/build/tasks/kernel.mk
+++ b/build/tasks/kernel.mk
@@ -68,6 +68,7 @@
 #                                          modules in vendor_overlay instead of vendor
 
 ifneq ($(TARGET_NO_KERNEL),true)
+ifneq ($(TARGET_NO_KERNEL_OVERRIDE),true)
 
 ## Externally influenced variables
 KERNEL_SRC := $(TARGET_KERNEL_SOURCE)
@@ -572,4 +573,5 @@ dtboimage: $(INSTALLED_DTBOIMAGE_TARGET)
 .PHONY: dtbimage
 dtbimage: $(INSTALLED_DTBIMAGE_TARGET)
 
+endif # TARGET_NO_KERNEL_OVERRIDE
 endif # TARGET_NO_KERNEL
-- 
2.25.1

