From c6833bbdfd617f7af4b3397dac1f4b22ba00211c Mon Sep 17 00:00:00 2001
From: naz664 <nazimnavas145@gmail.com>
Date: Tue, 13 Sep 2022 07:50:27 +0000
Subject: [PATCH 3/3] Remove duplicated adb-flag

---
 config/common.mk | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/config/common.mk b/config/common.mk
index 9e38607b..2fa1611c 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -37,7 +37,7 @@ include vendor/xdroid/config/xd_signed.mk
 include vendor/xdroidui/config.mk
 
 # xd. GMS
-include vendor/google/gms/config.mk
+#include vendor/google/gms/config.mk
 TARGET_SUPPORTS_GOOGLE_RECORDER ?= false
 TARGET_INCLUDE_STOCK_ARCORE ?= false
 TARGET_INCLUDE_LIVE_WALLPAPERS ?= false
@@ -127,3 +127,5 @@ DISABLE_EAP_PROXY := true
 # init.rc
 $(foreach f,$(wildcard vendor/xdroid/prebuilt/etc/init/*.rc),\
     $(eval PRODUCT_COPY_FILES += $(f):$(TARGET_COPY_OUT_SYSTEM)/etc/init/$(notdir $f)))
+
+TARGET_NO_KERNEL_OVERRIDE := true
-- 
2.25.1

