From 724be2c7d2b797b231506a64bbd7c308fb6c324c Mon Sep 17 00:00:00 2001
From: naz664 <nazimnavas145@gmail.com>
Date: Wed, 21 Dec 2022 12:07:08 +0000
Subject: [PATCH 2/3] remove dublicated adb-flag

---
 config/common.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config/common.mk b/config/common.mk
index 2199a9e..ccd5d2b 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -21,7 +21,7 @@ ifeq ($(TARGET_BUILD_VARIANT),eng)
 PRODUCT_SYSTEM_DEFAULT_PROPERTIES += ro.adb.secure=0
 else
 # Enable ADB authentication
-PRODUCT_SYSTEM_DEFAULT_PROPERTIES += ro.adb.secure=1
+# PRODUCT_SYSTEM_DEFAULT_PROPERTIES += ro.adb.secure=1
 
 # Disable extra StrictMode features on all non-engineering builds
 PRODUCT_SYSTEM_DEFAULT_PROPERTIES += persist.sys.strictmode.disable=true
-- 
2.34.1

