From 08f4fa9b83ff492f147068bb51b92091d1d5121a Mon Sep 17 00:00:00 2001
From: naz664 <nazimnavas145@gmail.com>
Date: Thu, 22 Dec 2022 14:27:11 +0000
Subject: [PATCH] configure ccache for builds

---
 build/envsetup.sh | 17 +++++++++++++++++
 1 file changed, 17 insertions(+)

diff --git a/build/envsetup.sh b/build/envsetup.sh
index 7350776..a1f15c6 100644
--- a/build/envsetup.sh
+++ b/build/envsetup.sh
@@ -1,3 +1,20 @@
+# Check and set ccache path on envsetup
+if [ -z ${CCACHE_EXEC} ]; then
+	ccache_path=$(which ccache)
+	if [ ! -z "$ccache_path" ]; then
+		mkdir /mnt/ccache
+		mount --bind ~/.ccache /mnt/ccache
+		export USE_CCACHE=1
+		export CCACHE_COMPRESS=1
+		export CCACHE_EXEC=/usr/bin/ccache
+		export CCACHE_DIR=/mnt/ccache
+		ccache -M 50G -F 0
+		echo "ccache found and CCACHE_EXEC has been set to : $ccache_path"
+	else
+		echo "ccache not found/installed!"
+	fi
+fi
+
 function __print_spark_functions_help() {
 cat <<EOF
 Additional sparkOS functions:
-- 
2.25.1

